galleries/examples/mplot3d/rotate_axes3d_sgskip.py

```diff
@@ -33,19 +33,19 @@
     angle_norm = (angle + 180) % 360 - 180
 
     # Cycle through a full rotation of elevation, then azimuth, roll, and all
-    elev = azim = roll = 0
+    azim = elev = roll = 0
     if angle <= 360:
         elev = angle_norm
     elif angle <= 360*2:
         azim = angle_norm
     elif angle <= 360*3:
         roll = angle_norm
     else:
-        elev = azim = roll = angle_norm
+        azim = elev = roll = angle_norm
 
     # Update the axis view and title
-    ax.view_init(elev, azim, roll)
-    plt.title('Elevation: %d°, Azimuth: %d°, Roll: %d°' % (elev, azim, roll))
+    ax.view_init(elev=elev, azim=azim, roll=roll)
```

Comment:
+/-0 on using kwargs here. The use of kwargs is meaningful with pure numbers to communicate meaning and order of the parameters, but we already get that through the variable names.

Reply:
It seems just repetitive now, and not longer serves a purpose, I'll get rid of it, change it back to the original (also in other places).